checker/src/main/java/org/checkerframework/checker/signedness/jdk.astub

```diff
@@ -94,6 +98,11 @@ class Long {
     public static @Unsigned long remainderUnsigned(@Unsigned long dividend, @Unsigned long divisor);
 }
 
+class Object {
+    protected @PolySigned Object clone() throws CloneNotSupportedException;
```

Comment:
You have used only one `@PolySigned`, and there is no occurrence in a binding site. 
You need to write `@PolySigned` on the receiver as well.

Reply:
Yes, you are right. On the other hand, if we have to use clone in Integer or String then we again have to define the function in the program. I think for the time being we should not annotate it.

Reply:
> if we have to use clone in Integer or String then we again have to define the function in the program.

Can you explain why?

Reply:
Like this-
import java.lang.*;
class Integer implements Cloneable{
	public Object clone()throws CloneNotSupportedException{
	return super.clone();  
	}
}
class ex2 {

		void play(){

		try{
		Integer ci=null;
		Integer b=(Integer)ci.clone();
		}
		catch(CloneNotSupportedException c){}
	}
}

Reply:
If we try to implement clone inn Integer than we can do by this.-
import java.lang.*;
class Integer implements Cloneable{
	public Object clone()throws CloneNotSupportedException{
	return super.clone();  
	}
}
class ex2 {

		void play(){

		try{
		Integer ci=null;
		Integer b=(Integer)ci.clone();
		}
		catch(CloneNotSupportedException c){}
	}
}

but here Integer is a class with a name "Integer". This Integer is not the same Integer which we want. I googled but I am unable to find anything on how to use clone with it.

Reply:
Now I understand that the issue is that Integer does not implement Cloneable, so myInteger.clone() could never work, regardless of annotations.  Thanks for the explanation.  Removing the annotation on Object.clone() seems fine, in this case.